src/main/java/seedu/address/logic/parser/AddPersonCommandParser.java

```diff
@@ -45,8 +49,10 @@ public AddPersonCommand parse(String args) throws ParseException {
                 ParserUtil.parseInternshipId(argMultimap.getValue(PREFIX_INTERNSHIP_ID).orElse(null));
 
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
+        Index linkIndex = ParserUtil.parseLinkIndex(Objects.requireNonNull(argMultimap
+                .getValue(PREFIX_LINK_INDEX).orElse(null)));
```

Comment:
parenthesis closure error resulting in null being passed to requireNonNull()
```suggestion
        Index linkIndex = ParserUtil.parseLinkIndex(Objects.requireNonNull(argMultimap
                .getValue(PREFIX_LINK_INDEX)).orElse(null));
```